eng/tools/generator/cmd/v2/common/generation.go

```diff
@@ -328,6 +328,11 @@ func (ctx *GenerateContext) GenerateForSingleRPNamespace(generateParam *Generate
 			}
 		}
 
+		log.Printf("Replace README.md module...")
+		if err = replaceReadmeModule(packagePath, generateParam.RPName, generateParam.NamespaceName, version.String()); err != nil {
```

Comment:
Only run this module when major version > 1. Or exit directly if major version > 1 within the function.

Reply:
It was released during refresh mgmt. The current version is v2.0.0-beta-1, but a stable v1.2.0 is required

Reply:
In that case, the module in README.md will not correspond